cmd/list_values.go

```diff
@@ -0,0 +1,163 @@
+	l "github.com/cloudposse/atmos/pkg/list"
+	fl "github.com/cloudposse/atmos/pkg/list/flags"
+	f "github.com/cloudposse/atmos/pkg/list/format"
+	"github.com/cloudposse/atmos/pkg/schema"
+	u "github.com/cloudposse/atmos/pkg/utils"
+)
+
+var (
+	ErrGettingCommonFlags    = errors.New("error getting common flags")
+	ErrGettingAbstractFlag   = errors.New("error getting abstract flag")
+	ErrGettingVarsFlag       = errors.New("error getting vars flag")
+	ErrInitializingCLIConfig = errors.New("error initializing CLI config")
+	ErrDescribingStacks      = errors.New("error describing stacks")
+)
+
+// Error format strings.
+const (
+	ErrFmtWrapErr = "%w: %v" // Format for wrapping errors.
+)
+
+// listValuesCmd lists component values across stacks.
+var listValuesCmd = &cobra.Command{
+	Use:   "values [component]",
+	Short: "List component values across stacks",
+	Long:  "List values for a component across all stacks where it is used",
+	Example: "atmos list values vpc\n" +
+		"atmos list values vpc --abstract\n" +
+		"atmos list values vpc --query '.vars'\n" +
+		"atmos list values vpc --query '.vars.region'\n" +
+		"atmos list values vpc --format json\n" +
+		"atmos list values vpc --format yaml\n" +
+		"atmos list values vpc --format csv",
+	Args: cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) != 1 {
+			log.Error("invalid arguments. The command requires one argument 'component'")
+			return
+		}
+
+		// Check Atmos configuration
+		checkAtmosConfig()
+		output, err := listValues(cmd, args)
+		if err != nil {
+			log.Error(err.Error())
+			return
+		}
+
+		u.PrintMessage(output)
+	},
+}
+
+// listVarsCmd is an alias for 'list values --query .vars'.
+var listVarsCmd = &cobra.Command{
+	Use:   "vars [component]",
+	Short: "List component vars across stacks (alias for 'list values --query .vars')",
+	Long:  "List vars for a component across all stacks where it is used",
+	Example: "atmos list vars vpc\n" +
+		"atmos list vars vpc --abstract\n" +
+		"atmos list vars vpc --max-columns 5\n" +
+		"atmos list vars vpc --format json\n" +
+		"atmos list vars vpc --format yaml\n" +
+		"atmos list vars vpc --format csv",
+	Args: cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		// Set the query flag to .vars
+		if err := cmd.Flags().Set("query", ".vars"); err != nil {
+			log.Error("failed to set query flag", "error", err, "component", args[0])
+			return
+		}
+		// Run the values command
+		listValuesCmd.Run(cmd, args)
+	},
+}
+
+func init() {
+	// Add common flags
+	fl.AddCommonListFlags(listValuesCmd)
+	if queryFlag := listValuesCmd.PersistentFlags().Lookup("query"); queryFlag != nil {
+		queryFlag.Usage = "Filter the results using YQ expressions"
+	}
+
+	// Add additional flags
+	listValuesCmd.PersistentFlags().Bool("abstract", false, "Include abstract components")
+	listValuesCmd.PersistentFlags().Bool("vars", false, "Show only vars (equivalent to --query .vars)")
```

Comment:
We support markdown.

```suggestion
	listValuesCmd.PersistentFlags().Bool("vars", false, "Show only vars (equivalent to `--query .vars`)")
```